src/conn/conn_cache_pool.c

```diff
@@ -154,7 +154,7 @@ __wt_cache_pool_config(WT_SESSION_IMPL *session, const char **cfg)
 	 *  - We are joining a cache pool for the first time (including
 	 *  creating the pool) - use the chunk size; that's the default.
 	 */
-	if (__wt_config_gets(session, &cfg[1],
+	if (__wt_config_gets(session, cfg,
```

Comment:
Can you explain this change? The code was trying to pull just the user-specified value and exclude the base configuration.